src/site/xdoc/tar.xml

```diff
@@ -129,14 +129,14 @@
         <p>Prior to Commons Compress 1.20 <code>TarArchiveInputStream</code> would recognize sparse
         file entries stored using the "oldgnu" format
         (<code>-&#x2d;sparse-version=0.0</code> in GNU tar) but not
-        able to extract them correctly. Starting with Commons Compress
+        able to extract them correctly. Starting with Commons Compress 1.21
```

Comment:
I think this should be 1.20. 
#87 introduced this feature and it's included in 1.20